index.md

```diff
@@ -90,11 +90,13 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
 | 3 | howto/port/configure-watchdog | [Configure the watchdog](https://discourse.ubuntu.com/t/configure-the-watchdog/26057)|
 | 3 | howto/port/install-system-app | [Install APK as a system app](https://discourse.ubuntu.com/t/install-an-apk-as-a-system-app/27086)|
 | 2 | howto/container/landing | [Work with containers](https://discourse.ubuntu.com/t/work-with-containers/24335) |
-| 3 | howto/container/launch | [Launch a container](https://discourse.ubuntu.com/t/launch-a-container/24327)|
+| 3 | howto/container/create | [Create a container](https://discourse.ubuntu.com/t/launch-a-container/24327)|
```

Comment:
Need to add a redirect (at the bottom of this file) from `howto/container/launch` to `howto/container/create`.